content/wasm-languages/assemblyscript.md

```diff
@@ -53,32 +53,34 @@ This section provides a basic example of building AssemblyScript from source.
 
 Set up the project like this:
 
-```console
+```bash
+$ mkdir typescript-example
```

Comment:
Actually the question about the "typescript" naming belongs here I guess - sorry!